### PR TITLE
Megachart data

### PR DESCRIPTION
**Pull request by gemfarmer**
Fixes issue(s) #1947, #1871 

[CircleCI](https://circleci.com/gh/18F/doi-extractives-data/tree/megachart-bug-killing-spree)

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/doi-extractives-data/megachart-bug-killing-spree/)

Changes proposed in this pull request:
- #1871  – Civil Penalties
- #1947 - Follow up oil gas funkiness
- added asterisk, note in other revenues national revenue by type
- added Easement and Right of Way data into the offshore revenue regional rollups

/cc @meiqimichelle

### CODE REVIEW

File: test/data/revenue.js [hunk 1]
```diff
@@ -117,62 +124,111 @@ describe('revenues by type', function() {
...
+      };
+
+      loadPivot(pivotSource, function(error, rows) {
+        rows.forEach(testRow);
+        done();
+      });
+
+
+    });
+  })
```
Comment (gemfarmer) [inline test/data/revenue.js]: fixed with a3ca8de

**Comment by gemfarmer** [comment]
With that Easement and Right of the Way madness out of the way I think this baddy is ready for review, @meiqimichelle 🎉

**Comment by meiqimichelle** [comment]
On the national profile page, the megachart includes civil penalties etc, but the All Commodities bar chart doesn't so the totals are off. Methinks we should include these new additions in the bar totals as well. ie, the $7,... on the top left and bottom left aren't the same:

screenshot 2016-09-16 15 13 41

**Comment by meiqimichelle** [comment]
Otherwise, I think we're good since Chris checked the dataaaa! Yeeeeaaaaayyyy! Should we keep that other issue here, or pull it into our other total-mismatch issue, #1973 @gemfarmer ?

**Comment by gemfarmer** [comment]
@meiqimichelle let's pull it into #1973

**Comment by gemfarmer** [comment]
Ok. So all satellite bugs related to this have been accounted for, from what I can tell. I say it is good to go @meiqimichelle. Anything else I haven't noticed?

**Pull request merged**